extensions/Worker.Extensions.ServiceBus/src/ServiceBusExtensionStartup.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Azure.Functions.Worker;
+
+public class ServiceBusExtensionStartup : WorkerExtensionStartup
+{
+    public override void Configure(IFunctionsWorkerApplicationBuilder applicationBuilder)
+    {
+        if (applicationBuilder == null)
+        {
+            throw new ArgumentNullException(nameof(applicationBuilder));
+        }
+
+        applicationBuilder.Services.AddAzureClientsCore(); // Adds AzureComponentFactory
```

Comment:
Do we want logs forwarded or no?   This form won't start the forwarder, right?

Reply:
Good question - the logs will still be forwarded on the extension side so I would assume users would still see them. I was mostly just copying the pattern from https://github.com/Azure/azure-functions-dotnet-worker/pull/1108.